.github/workflows/node.js.yml

```diff
@@ -0,0 +1,60 @@
+name: Unit Tests
```

Comment:
Maybe this workflow should be title jest instead of node

Reply:
```suggestion
name: Jest Tests
```

Reply:
sure